tests/libcxx/expected_results.txt

```diff
@@ -0,0 +1,1023 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+

```

Comment:
I am concerned that we now have two lists of skipped tests for libcxx; this seems like it will get desynced easily. Is there some way to rely on a single list, or possibly a main list with a small secondary list of Python-runner-specific skips? Ideally, someone with a libcxx update that passes checks on GitHub should automatically pass on MSVC-internal and vice versa.

Reply:
I see that you and Casey talked about this and decided to go with duplicated file lists for the time being. Ok.